sympy/assumptions/sathandlers.py

```diff
@@ -300,6 +300,9 @@ def register_fact(klass, fact, registry=fact_registry):
     (Mul, Equivalent(Q.zero, AnyArgs(Q.zero))),
     (MatMul, Implies(AllArgs(Q.square), Equivalent(Q.invertible, AllArgs(Q.invertible)))),
     (Add, Implies(AllArgs(Q.positive), Q.positive)),
+    (Add, Implies(AllArgs(Q.positive), ~Q.negative)),
```

Comment:
We could have ``Implies(Q.positive, ~Q.negative)``. Or perhaps ``Implies(Q.positive, Q.nonnegative)`` and ``Implies(Q.nonnegative, ~Q.negative)``. Then this line would follow from the line above.